setup.py

```diff
@@ -36,6 +36,7 @@
           'rx',
           'psutil;platform_system=="Linux"',
           'futures;python_version<"3.3"',
+          'ptyprocess'
```

Comment:
Could you change this line to:
`'ptyprocess;platform_system=="Linux"'`

Although it doesn't matter as Linux is only supported platform, but psutil has the same requirement.